keyboards/skyloong/gk87/bl/bl.c

```diff
@@ -0,0 +1,91 @@
+//  Copyright 2023 NaturalZh (@NaturalZh)
+// SPDX-License-Identifier: GPL-2.0-or-later
+#include "quantum.h"
+
+_Bool DIS_BRETH = 0;
```

Comment:
Nit-pick: generally, we use all upper case for preprocessors, and snake_case for code.

Also, you can use "bool" rather than `_Bool` hear.

In both cases, they are fine, but just bother me.

Reply:
OK，I modified it . 
Thanks

Reply:
Not good ;(
Change good later, I try to compile the keyboard, but the compiler told me: keyboards/skyloong gk87 / bl/bl. C: 5-1: error: unknown type name '_bool'; did you mean '_Bool'?